Cargo.toml

```diff
@@ -120,7 +120,8 @@ bitflags          = "2.3.1"
 bpaf              = { version = "0.9.5", features = ["derive"] }
 countme           = "3.0.1"
 dashmap           = "5.4.0"
-ignore            = "0.4.20"
+globset           = "0.4.14"
+ignore            = "0.4.21"
```

Comment:
DO we still need the `ignore` crate?

Reply:
Yes, we use it to parse the `.gitignore` contents when we enable the VCS integration. `ignore` also has a [`::gitignore`](https://docs.rs/ignore/latest/ignore/gitignore/index.html) module that does the parsing and exposes utility functions.